src/display.ts

```diff
@@ -311,6 +331,28 @@ const display = (
   ];
 
   for (const info of featureInfos) {
+    if (face.aging && face.aging.enabled) {
```

Comment:
Similar to what I wrote about hair... maybe switch this around, so some facial lines are just never (or more rarely) included in the base face object, and then apply them here (maybe again with deterministic randomness). I think that would make it easier to have aging appear in all faces. Since currently, when I'm playing around with this, a lot of faces are not impacted by aging.

Reply:
> My understanding is that you implemented this version in BBGM and you didn't see many aged faces?

No, just playing around in the editor UI. But it's the same thing, BBGM generates random faces the same way the editor does.

> Also, I very arbitrarily chose different ages for different lines to appear. They could easily be too high. There could also be a variable called maturity that can span +/- 3 or something which can add an element of variation across the players on when they reach these arbitrary age thresholds.

Probably a good idea. On second thought, what I wrote before about using deterministic random numbers to determine aging might be kind of annoying, because then users wouldn't be able to control it...

lol complicated stuff the more you think about it!